app/views/members/edit.html.erb

```diff
@@ -1,5 +1,7 @@
-<h2>Edit Member</h2>
+<div class="col-10 p-3">
+  <h2>Edit Member</h2>
```

Comment:
pass heading and submit button text to form in locals and do everything over there